src/main/webapp/WEB-INF/redirects.properties

```diff
@@ -30,6 +30,12 @@
 /docs/ref/javaee/=/docs/ref/javaee/8/
 /docs/ref/microprofile/=/docs/ref/microprofile/3.3/
 /javadocs/*=/docs/ref/javadocs/
+/docs/ref/feature/featureOverview.html=/docs/latest/feature/feature-overview.html
```

Comment:
this is fine for now, but we need to open an issue to change this such that we don't have to update this every release.  

So adding support for something like this:
/docs/(.*)$/reference/feature/featureOverview.html=/docs/$1/reference/feature/feature-overview.html

In other words, I want to add support for regex in our redirects, or add variable support to our existing one (performance will need to be considered).